lineage/figures/figure16.py

```diff
@@ -0,0 +1,185 @@
+    ax[i].scatter(x, paramTrues[:, 1, 4], marker="_", alpha=0.5, label="State 2")
+    ax[i].legend()
+    ax[i].set_title(r"Gamma $\theta$ G2")
+    ax[i].grid(linestyle="--")
+    ax[i].tick_params(axis="both", which="major", grid_alpha=0.25)
+
+    i += 1
+    ax[i].set_xlabel(xlabel)
+    ax[i].set_ylim(bottom=0, top=101)
+    ax[i].scatter(x, accuracies, c="k", marker="o", label="Accuracy", edgecolors="k", alpha=0.25)
+    ax[i].set_ylabel(r"Accuracy [\%]")
+    ax[i].axhline(y=100, linestyle="--", linewidth=2, color="k", alpha=1)
+    ax[i].set_title("State Assignment Accuracy")
+    ax[i].grid(linestyle="--")
+    ax[i].tick_params(axis="both", which="major", grid_alpha=0.25)
+
+    i += 1
+    ax[i].set_xlabel(xlabel)
+    ax[i].set_ylim(bottom=0, top=max(tr) + 0.2)
+    ax[i].scatter(x, tr, c="k", marker="o", edgecolors="k", alpha=0.25)
+    ax[i].set_ylabel(r"$||T-T_{est}||_{F}$")
+    ax[i].axhline(y=0, linestyle="--", linewidth=2, color="k", alpha=1)
+    ax[i].set_title("Transition Matrix Estimation")
+    ax[i].grid(linestyle="--")
+    ax[i].tick_params(axis="both", which="major", grid_alpha=0.25)
+
+    i += 1
+    ax[i].set_xlabel(xlabel)
+    ax[i].set_ylim(bottom=0, top=max(pii) + 0.2)
+    ax[i].scatter(x, pii, c="k", marker="o", edgecolors="k", alpha=0.25)
+    ax[i].set_ylabel(r"$||\pi-\pi_{est}||_{2}$")
+    ax[i].axhline(y=0, linestyle="--", linewidth=2, color="k", alpha=1)
+    ax[i].set_title("Initial Probability Matrix Estimation")
+    ax[i].grid(linestyle="--")
+    ax[i].tick_params(axis="both", which="major", grid_alpha=0.25)
+
+def makeFigure():
+    """
+    Makes figure 16.
+    """
+
+    # Get list of axis objects
+    ax, f = getSetup((12, 6), (2, 4))
+
+    figureMaker(ax, *accuracy(), xlabel=r"Cells in State 0 [$\%$]")
+
+    subplotLabel(ax)
+
+    return f
+
+
+def accuracy():
+    """
+    Calculates accuracy and parameter estimation
+    over an similar number of cells in a lineage for
+    a uncensored two-state model but differing state distribution.
+    We increase the proportion of cells in a lineage by
+    fixing the Transition matrix to be biased towards state 0.
+    """
+
+    # Creating a list of populations to analyze over
+    list_of_Ts = [np.array([[i, 1.0 - i], [i, 1.0 - i]]) for i in np.linspace(0.1, 0.9, num_data_points)]
+    list_of_populations = []
+    list_of_fpi = []
+    list_of_fT = []
+    list_of_fE = []
+    for T in list_of_Ts:
+        population = []
+
+        good2go = False
+        while not good2go:
+            tmp_lineage = LineageTree(pi, T, E, max_desired_num_cells)
+            good2go = lineage_good_to_analyze(tmp_lineage)
+
+        population.append(tmp_lineage)
+
+        # Adding populations into a holder for analysing
+        list_of_populations.append(population)
+        list_of_fpi.append(pi)
+        list_of_fT.append(T)
+        list_of_fE.append(E)
+
+    return commonAnalyze(list_of_populations, xtype="length", list_of_fpi=list_of_fpi)
```

Comment:
Yea so for this figure (I think this is figure 6 right), each lineage has the same number of cells (which is why all the data points are on the same number). The only thing that differs from each lineage is the transition matrix (line 164). I would recommend copying like Figure 5 (GROUPS of CENSORED lineages) or Figure 3 (GROUPS of UNCENSORED lineages) and then using the state distribution.